.vscode/launch.json

```diff
@@ -164,6 +164,38 @@
         "services/validation/test/truffle-example-incorrect.zip"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Validation service (truffle project - directory)",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/services/validation/build/index.js",
+      "preLaunchTask": "tsc: build - services/validation/tsconfig.json",
+      "outFiles": [
+        "${workspaceFolder}/services/validation/build/**/*.js"
+      ],
+      "args": [
+        "services/validation/test/truffle-example"
+      ]
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Validation service (incorrect truffle project - directory)",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/services/validation/build/index.js",
+      "preLaunchTask": "tsc: build - services/validation/tsconfig.json",
+      "outFiles": [
+        "${workspaceFolder}/services/validation/build/**/*.js"
+      ],
+      "args": [
+        "services/validation/test/truffle-example-incorrect"
```

Comment:
Rename `truffle-example directory` to` truffle-example-incorrect` and it should be fine.

Reply:
Caused by issues in .gitignore; now resolved.